### ISSUE TITLE
Add feature to delete built-in plugins?

### ISSUE THREAD

**Issue by Xane123** [MaximumADHD/Roblox-Studio-Mod-Manager#145]
I don't like the built-in "UI Editor" plugin (and find Audio Discovery basically useless), so I like to remove the files on each update using my own batch script. However, as I can't also set FFlags using that method, I've returned to using Mod Manager, which I've found has a folder that lets me override any built-in sound, texture, or built-in plugin.

There's one problem with it, though; I don't think there's a way to tell Mod Manager to _delete_ certain plugins' .rbxm files, only replacing them with a new file. I tried replacing them with mostly empty files that contain "<roblox!>< /roblox>", but when I open my game or test it in Studio, I get a ton of errors about my replacements:
![image](https://user-images.githubusercontent.com/17244146/191618709-e14ba822-3fc7-4fbf-9482-ea539074d14e.png)
(These errors don't appear at all when I delete the files instead.)

So, to sum this up, Mod Manager has a "mod folder" for replacing built-in Studio files, but there's no way to completely remove files. I think this would be a good feature to add; Maybe if a file that's named and placed in the same folder within [the mod folder]/BuiltInPlugins that is 0 bytes, it could delete the original Studio file?

**Comment by DiamondBor** [comment]
Maybe just remove all code from the plugin script and put it back in the folder so it wont appear or be replaced

**Comment by Xane123** [comment]
Hmm... That probably would get rid of some of the errors, but I think the ones that mention its "signature" would still appear because for some reason, Studio expects all of the built-in plugins to be signed.

Removing the files completely gets rid of all of the errors relating to them, so I still think it would be a good feature.

**Comment by MaximumADHD** [comment]
Funny enough this does exist.
You just need to add a file to the corresponding folder called `DELETE PluginName.rbxm` and it'll be auto-deleted.
